projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_dag.py

```diff
@@ -109,7 +117,16 @@ def default_serialization(o: Any) -> Any:
 
     def _start_job(self, node):
         try:
-            self._job_executor.start_job(node)
+            if (
+                len(self._job_executor.get_currently_running_jobs())
+                > self.max_concurrent_starting_jobs
```

Comment:
Probably should be `>=`
```suggestion
                len(self._job_executor.get_currently_running_jobs())
                >= self.max_concurrent_starting_jobs
```

Reply:
Yep, changed it.